script.js

```diff
@@ -0,0 +1,37 @@
+
+import {displayGrid, resetDisplay, setDisplayedGridSize} from "./display.js";
+import {activateFlaggingButton, clickToReveal, startGame} from "./gamePlay.js";
+import Grid from "./Grid.js";
+import Game from "./Game.js";
+
+const mineSweepGrid = new Grid(9, 10);
```

Comment:
9x10 seems a bit odd to start with, I'd think it should be square but not the biggest deal
```suggestion
const mineSweepGrid = new Grid(9, 9);
```

Reply:
refers to the number of mines, it's always square